README.md

```diff
@@ -51,10 +51,14 @@ If you want to deploy this application locally on your computer you will need [N
 Download this repository and after the download you need to add an .env file on the root folder of the application with the following API keys you can get for free:
```

Comment:
It would be a good idea (for other ticket) to create an `.env.example` file so the person that donwloads this project will only need to remove the `.example` and fill the vars.

Reply:
Opened new issue for this